sympy/functions/elementary/exponential.py

```diff
@@ -524,9 +524,6 @@ def eval(cls, arg, base=None):
                 return S.Infinity
             elif arg is S.NaN:
                 return S.NaN
-            elif arg.is_Rational:
```

Comment:
What about making this 
```
elif arg.is_Rational and arg.p == 1:
    return - cls(arg.q)
```
So log(1/3) -> -log(3) and log(3/5, 1/5) -> -log(3)/log(5) + 1` instead of `log(3)/log(1/5) + 1`?

Reply:
That looks like a good idea to me.

Reply:
@jksuom @smichr I think `evaluate=False` should also be added
And should I remove the addition of `_eval_expand_log`, or that change is correct?

Reply:
I suspect it's safe to make evaluate False.